templates/4_Post_Clustering.ipynb

```diff
@@ -24,7 +24,8 @@
     "import numpy as np\n",
```

Comment:
<p>Will you add a link to this notebook (and the others) that import the example dataset with a link to the readme section, saying that there's more information there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>